index.html

```diff
@@ -2406,6 +2634,1425 @@ <h2>
         </p>
       </div>
     </section>
+    <section id='automation'>
+      <h2>
+        Automation
+      </h2>
+      <p>
+        For the purposes of [=user agent=] automation and application testing, this document defines extensions to the [[WebDriver-BiDi]] expecification.
+        It is OPTIONAL for a [=user agent=] to support them.
+      </p>
+      <p>
+        Each [=top-level traversable=] has a <dfn>simulated gamepad map</dfn> which is an [=ordered map=] of [=version 4 UUIDs=] to [=gamepad input sources=].
```

Comment:
I think the UUID key it sufficient and we don't need to have it ordered map.
```suggestion
        Each [=top-level traversable=] has a <dfn>simulated gamepad map</dfn> which is a [=map=] of [=version 4 UUID=] and [=gamepad input source=].
```